### PR TITLE
Made components using new prop-types npm package

### PR DESCRIPTION
**Pull request by fragsalat**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -97,6 +98,7 @@
   },
   "peerDependencies": {
     "preact": "7.1.0",
-    "preact-compat": "3.13.1"
+    "preact-compat": "3.13.1",
```
Comment (faboweb) [inline package.json]: why do we need preact-compat?

Comment (faboweb) [inline package.json]: Do I need to add preact-compat to my setup?

File: src/ws-dropdown/dropdown-menu.js [hunk 2]
```diff
@@ -182,7 +182,9 @@ export class DropdownMenu extends Component {
         this.showChild(data);
         break;
       case 'change':
-        this.clearSelections();
+        if (this.context.multiple) {
```
Comment (faboweb) [inline src/ws-dropdown/dropdown-menu.js]: could you maybe add a comment what this means?

File: package.json (hunk 1 above)
Comment (fragsalat) [inline package.json]: it is a peerDependency which means it is not installed automatically but you can install it by yourself. In general there should also be react in this list and you could choose which peer u want to fulfill.

So no you don't.

File: src/ws-dropdown/dropdown-menu.js (hunk 2 above)
Comment (fragsalat) [inline src/ws-dropdown/dropdown-menu.js]: Will follow

**Pull request merged**